hs628_tools/random-forest.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+
+def rf(features, target, model='classifier'):
```

Comment:
If your model is defined as classifier in your variables passed in, then your if/else wont work.